applications/asset_tracker_v2/boards/nrf9160dk_nrf9160ns.conf

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
File needs to be mentioned in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/README.html#configuration-files